client/client.go

```diff
@@ -85,6 +85,12 @@ func (c *Client) ListTasks(ctx context.Context, req *tes.ListTasksRequest) (*tes
 		addString(v, "state", req.State.String())
 	}
 
+	if req.GetTags() != nil {
```

Comment:
You don't need this. It's ok to range over a nil map: https://play.golang.org/p/e-ynTY8LdVx